src/components/Editor.js

```diff
@@ -150,6 +152,30 @@ const Editor = React.createClass({
     }
   },
 
+  toggleBreakpointDisabledStatus(line) {
```

Comment:
Hmm, I can see what you're saying. It is also consistent with `toggleBreakpoint` which is right above, which makes sense. 

Thanks for being patient @shlomif.